.github/workflows/fossa.yml

```diff
@@ -0,0 +1,33 @@
+name: FOSSA analysis
+on:
+  push:
+    branches:
+      - '*'
+    paths-ignore:
+      - '**.md'
+      - '**/NOTICE'
+      - '**/LICENSE'
+  pull_request:
+    branches:
+      - 'develop'
+    paths-ignore:
+      - '**.md'
+      - '**/NOTICE'
+      - '**/LICENSE'
```

Comment:
You are advised to ignore the integration test.

Reply:
it is necessary